server/channels/app/helper_test.go

```diff
@@ -310,10 +309,9 @@ func (th *TestHelper) CreateTeam() *model.Team {
 		Type:        model.TeamOpen,
 	}
 
-	var err *model.AppError
-	if team, err = th.App.CreateTeam(th.Context, team); err != nil {
-		panic(err)
-	}
+	var appErr *model.AppError
+	team, appErr = th.App.CreateTeam(th.Context, team)
```

Comment:
Does this also work?
```suggestion
	team, appErr := th.App.CreateTeam(th.Context, team)
```